telegram/chatmember.py

```diff
@@ -242,7 +250,16 @@ def de_json(cls, data: Optional[JSONDict], bot: 'Bot') -> Optional['ChatMember']
         data['user'] = User.de_json(data.get('user'), bot)
         data['until_date'] = from_timestamp(data.get('until_date', None))
 
-        return cls(**data)
+        _class_mapping: Dict[str, Type['ChatMember']] = {
```

Comment:
I know that this setup comes from me, but at second thought it needs fine tuning: Because the subclasses inherit this behavior, currently `ChatMemberOwner.de_json` does not always return a `ChatMemberOwner` instance, which is probably unexpected. So we could either override in the subclasses or just insert an `if cls is ChatMember:` here 🤔 same for `BotCommandScope` …

The docstring could also be updated like for BotCommandScope as we're doing a bit more magic here :)

Reply:
I would prefer second approach, as it is much cleaner to have just one `if` check in base class than overriding same method in all subclasses..